src/ansys/fluent/core/session_solver.py

```diff
@@ -344,7 +357,7 @@ def __call__(self):
 
     def __getattribute__(self, item: str):
```

Comment:
This needs better typing desperately as it will allow you use basically anything at type time currently